### DISCUSSION TITLE
Welcome to partR2 Discussions!

### DISCUSSION THREAD

**Discussion by mastoffel** [mastoffel/partR2#2] [General]
Dear partR2 users,

this is a more informal section to ask for features or discuss anything around the package which doesn't fit in 'Issues'.

**Comment by David0087** [comment]
Hi

I have just read about your package and would like to use partR2 for evaluating a multilevel model fitted using nlme package. Is it working with nlme objects (rather than lme4)?

Thank you very much

**Reply by mastoffel** [reply]
Hi David,

partR2 currently only works with lme4 unfortunately. Can I ask what kind of model you are using in nlme?

Cheers,
Martin

**Reply by David0087** [reply]
Hi Martin,

Thanks for your answer, I am planning a very simple model of linear mixed effect, but my point is that after fitting the model I need to use the function IndvPred_lme from the JMBayes package so I need an lme object.
